C_routing.md

```diff
@@ -301,58 +300,69 @@ end
 
 Note that Phoenix will assume that the path we set ought to begin with a slash, so `scope "/admin" do` and `scope "admin" do` will both produce the same results.
 
-Note also, that the way this scope is currently defined, we need to fully qualify our controller name, `HelloPhoenix.Admin.ReviewController`. We'll fix that in a minute.
+Note also, that the way this scope is currently defined, we need to fully qualify our controller name, `Admin.ReviewController`. We'll fix that in a minute.
```

Comment:
Actually, this is not correct. The fully qualified name is "HelloPhoenix.Admin.ReviewController"